### ISSUE TITLE
PowerMonitor "resume" and "suspend" event send twice on macOS

### ISSUE THREAD

**Issue by sevilen** [electron/electron#24803]
### Preflight Checklist

* [x ] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/master/CONTRIBUTING.md) for this project.
* [ x] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x ] I have searched the issue tracker for an issue that matches the one I want to file, without success.

### Issue Details
When I upgrade the electron version 6.1.9 to 8.4.0, the PowerMonitor module "resume" and "suspend" event send twice on macOS.

useage:
```
app.on("ready", () => {
    const { powerMonitor } = require('electron')
    
    powerMonitor.on('suspend', () => {
        mainLog.info('system suspend.')
        mainWindow.webContents.send('system-status', 'suspend');
    });

    powerMonitor.on('resume', () => {
        mainLog.info('system resume.')
        mainWindow.webContents.send('system-status', 'resume');
    });
});
```
electron-log, main.log:
```
[2020-07-31 11:29:41.926] [info] system suspend.
[2020-07-31 11:29:41.933] [info] system suspend.
[2020-07-31 11:30:20.169] [info] system resume.
[2020-07-31 11:30:21.963] [info] system resume.
```
* **Electron Version:**
  * 8.4.0

* **Operating System:**
  * macOS 10.15.6
* **Last Known Working Electron version:**
  * 8.4.0

### Expected Behavior

just one times "resume" and "suspend" event.
### Actual Behavior

i receive twice same events.

**Comment by codebytere** [comment]
https://github.com/electron/electron/blob/4290555a0d9dacc4ba46650cd2b1d5baca0e2467/shell/browser/api/electron_api_power_monitor_mac.mm#L63-L73

is what mediates that code - we can likely just emit once regardless of # of emitters. I'll try to dial that up soon.

**Comment by codebytere** [comment]
@sevilen I opened a PR - should be resolved soon.

**Comment by sevilen** [comment]
@codebytere thanks a lot.

**Comment by DangerBlack** [comment]
This still affect electron 9.1.1

**Comment by codebytere** [comment]
Yes, that would be because this fix was released in 9.2.0.
